self_supervised_pretraining/README.md

```diff
@@ -0,0 +1,38 @@
+# Self-Supervised Pretraining Tutorial
+
+Authored by Vishwesh Nath (a.k.a Vish)
+
+This directory contains two scripts. The first script 'ssl_script_train.py' generates
+a good set of pre-trained weights using unlabeled data with self-supervised tasks that
+are based on augmentations of different types. The second script 'ssl_finetune_train.py' uses
+the pre-trained weights generated from the first script and finetunes on a fully supervised task.
+
+### 1.Data
+Pretraining Dataset: The TCIA Covid-19 dataset was used for generating the pretrained weights.
+The dataset contains a total of 771 3D CT Volumes. The volumes were split into training and validation sets
+of 600 and 171 3D volumes correspondingly. PUT LINKS & CITATIONS HERE
+
+Finetuning Dataset: The dataset from Beyond the Cranial Vault Challenge 2015 hosted at MICCAI was used as a
+fully supervised Finetuning task on the pre-trained weights. The dataset contains of 30 3D Volumes with annotated labels
+for 13 different organs. PUT LINKS & CITATIONS HERE
+
+### 2. Network Architectures
```

Comment:
I think the items 2 - 5 are still not updated?

Thanks.

Reply:
Yes, the objective for now is to outline all broad things that need to be covered. It will be addressed.

Reply:
These items have been updated.